dashboard/app/models/levels/blockly.rb

```diff
@@ -89,6 +89,12 @@ class Blockly < Level
   # DCDO key for turning this feature on or off.
   BLOCKLY_I18N_IN_TEXT_DCDO_KEY = 'blockly_i18n_in_text'.freeze
 
+  def summarize_for_lab2_properties(script)
```

Comment:
Trying to refresh my memory, thinking out loud a bit here - so we need `script` for caching purposes when calling `localized_blockly_level_options`, right? However, the only value that we currently grab from it is `sharedBlocks`. Looking at that file again, it seems like because `shared_blocks` simply calls `Blocks.for(type)` that shared blocks should be the same for all levels with the same block pool...so I'm wondering if instead we should just expose a separate, cacheable endpoint for fetching shared blocks for a given pool type? I also think there are some quirks in our caching system where certain scripts are or aren't cached, but not sure if that applies here.

@breville @molly-moen any thoughts here on fetching shared blocks for lab2 labs, thinking generally about blockly labs in Lab2 as a whole?

Reply:
> so we need script for caching purposes when calling localized_blockly_level_options, right? However, the only value that we currently grab from it is sharedBlocks. Looking at that file again, it seems like because shared_blocks simply calls Blocks.for(type) that shared blocks should be the same for all levels with the same block pool

Yes - sounds right to me!

> so I'm wondering if instead we should just expose a separate, cacheable endpoint for fetching shared blocks for a given pool type?

Interesting! Again - open to trying out. Are you thinking we should do this in this PR or in a separate PR?

Reply:
I think exposing and endpoint for shared blocks makes sense!

Reply:
Recapping our discussion offline -
I think eventually breaking out shared blocks and other similar properties into a separate endpoint would be great, but definitely doesn't have to be this PR since I think we'll need to look into this more. We'd want to look at blockly level data holistically and determine which properties do actually change per level and which properties remain constant per app type.